src/config.rs

```diff
@@ -493,7 +490,7 @@ impl Default for DefaultCfConfig {
             max_write_buffer_number: 5,
             min_write_buffer_number_to_merge: 1,
             max_bytes_for_level_base: ReadableSize::mb(512),
-            target_file_size_base: ReadableSize::mb(8),
+            target_file_size_base: ReadableSize::mb(128),
```

Comment:
I think the naming here is trickier than it seems. So the semantics of this config actually changes as we toggle the compaction guard, which could cause serious confusion. Maybe we can remove `compaction_guard_min_output_file_size` and implicitly takes the old `target_file_size_base` value as `compaction_guard_min_output_file_size` internally when guard is on? As for the maximum size limit we can keep the `compaction_guard_max_output_file_size` with less advertising.

Reply:
good suggestion.

Reply:
I think I'll keep the meaning and default value for `target_file_size_base`, but use `compaction_guard_max_output_file_size` to override it when compaction guard is enabled.